src/macho.rs

```diff
@@ -53,6 +53,13 @@ pub const CPU_TYPE_POWERPC64: u32 = CPU_TYPE_POWERPC | CPU_ARCH_ABI64;
 pub const CPU_SUBTYPE_MASK: u32 = 0xff00_0000;
 /// 64 bit libraries
 pub const CPU_SUBTYPE_LIB64: u32 = 0x8000_0000;
+/// pointer authentication with versioned ABI
+pub const CPU_SUBTYPE_PTRAUTH_ABI: u32 = 0x8000_0000;
```

Comment:
Did they use the same value as lib64?

Reply:
Yes. I don't know what distinguishes between them.